db/mockdata.py

```diff
@@ -0,0 +1,159 @@
+"""
```

Comment:
Lets go with api-tools.py.

If you need to use a non-whitespace separator, please use - and not _. It makes it easier to read overall.